content/features/featuresDeepDive/particles/particle_system/flowMap.md

```diff
@@ -0,0 +1,44 @@
+# Flow maps
+
+Starting from Babylon.js v8.6, you can provide a flow map to control how your particles behave.
+All particle systems support that feature.
+
+Here is a complete example to try:
+
+<Playground id="#5DM02T#7" title="Flow map" description="Complete example of a particle flow map."/>
+
+# Definition
+
+A flow map is a screen-aligned image that is used to control the direction and intensity of forces applied to particles based on their position on the screen, not in world space. 
+This means the flow behaves consistently relative to the viewport, regardless of camera movement or rotation.
+
+The flow map is a standard texture where each pixel encodes a 3D direction vector and strength using a Color4 format:
+
+Red (R): X-component of the flow direction (0 → left, 1 → right, 0.5 → no movement)

```

Comment:
We should be more explicit about directions using screen left, screen right, screen up, screen down, towards screen, and away from screen to eliminate any potential confusion.